### PR TITLE
Portfolio Accessibility

### PR DESCRIPTION
**Pull request by zunairkhan811**
Accessibility Points:

- Page titles --- no issues found.
- Image text alternatives --- no issues found.
- Text headings  --- The heading h2 is missing after h1 or h3 should be converted to h2 at following lines

1. https://github.com/zunairkhan811/Zunair-Desktop-Responsive-Portfolio/blob/2239838e50998adc9ecb89837119058d66d02895/index.html#L115-L128
2. https://github.com/zunairkhan811/Zunair-Desktop-Responsive-Portfolio/blob/2239838e50998adc9ecb89837119058d66d02895/index.html#L290-L365

- Color contrast --- The following lines have a contrast ratio of less than 4.5 

1. https://github.com/zunairkhan811/Zunair-Desktop-Responsive-Portfolio/blob/2239838e50998adc9ecb89837119058d66d02895/index.html#L147-L279
2. https://github.com/zunairkhan811/Zunair-Desktop-Responsive-Portfolio/blob/2239838e50998adc9ecb89837119058d66d02895/index.html#L102-L104
4. https://github.com/zunairkhan811/Zunair-Desktop-Responsive-Portfolio/blob/2239838e50998adc9ecb89837119058d66d02895/index.html#L304-L306
5. https://github.com/zunairkhan811/Zunair-Desktop-Responsive-Portfolio/blob/2239838e50998adc9ecb89837119058d66d02895/index.html#L91-L96
- Resize --- At 105% zoom text works fine but above this 105% all texts Overlap each other.
- Interaction --- no issues found.
- Moving content ---no issues found.
- Multimedia --- no issues found.
- The basic structure of the page --- no issues found.

### CODE REVIEW

**Review by Kamba56 — COMMENTED** [review]
Hi @zunairkhan811 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **doesn't have the commits of your partner.** this is a pair programming exercise, kindly work on it with your partner, I should be able to see commits from your partner.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: description.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Portfolio: ACCESSIBILITY
+
```
Comment (bobb-Rob) [inline description.txt:2]: - [x] To improve the page accessibility using keyboard navigation, the hamburger icon should be an actually icon wrapped in an `a` tag or `button` tag. These two elements can be accessed with a keyboard, usually the `tab` button.
Please fix this. 

image

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @zunairkhan811 and @Festy1 , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: description.txt (hunk 1 above)
Comment (zunairkhan811) [reply description.txt:2]: Add &lt;a&gt; tag around menu icon

**Review by binhussen — APPROVED** [review]
Hi @zunairkhan811, @Festy1  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**